test/extended/openstack/cloud-provider-config.go

```diff
@@ -46,23 +46,23 @@ var _ = g.Describe("[sig-installer][Suite:openshift/openstack] The Openshift", f
 			}
 
 			internalConfigMaps := []config_params{
-				config_params{
+				{
 					namespace: "openshift-config-managed",
 					name:      "kube-cloud-config",
 					key:       "cloud.conf",
-					skip:      nil,
+					skip:      []string{"use-octavia"},
 				},
-				config_params{
+				{
 					namespace: "openshift-cloud-controller-manager",
 					name:      "cloud-conf",
 					key:       "cloud.conf",
-					skip:      []string{"secret-name", "secret-namespace"},
+					skip:      []string{"secret-name", "secret-namespace", "use-octavia"},
 				},
-				config_params{
+				{
 					namespace: "openshift-cluster-csi-drivers",
 					name:      "cloud-conf",
 					key:       "cloud.conf",
-					skip:      []string{"secret-name", "secret-namespace"},
+					skip:      []string{"secret-name", "secret-namespace", "use-octavia"},
```

Comment:
is use-octavia present on the cloud-conf of openshift-cluster-csi-drivers?

Reply:
Yes, it does at the moment. The [cloud.conf transformation in cinder-csi-driver-operator](https://github.com/openshift/openstack-cinder-csi-driver-operator/blob/bf20cdb79fcc9c267eab4604ab9291d413ceff1f/pkg/controllers/config/configsync.go#L108) keep existing sections, in this case the `[LoadBalancer]` one.